webdriver/tests/get_element_tag_name/user_prompts.py

```diff
@@ -7,9 +7,9 @@ def read_global(session, name):
     return session.execute_script("return %s;" % name)
 
 
-def get_tag_name(session, element_id):
+def get_tag_name(session, webdriver_element_id):
```

Comment:
All other tests use `element_id` here. Can you change that please?

Reply:
Happy to. I was simply worried that the formatter would get confused between the token name and variable name. I should’ve tried it myself first.